### ISSUE TITLE
The tool `agent_control` could show a malformed `Client version` field

### ISSUE THREAD

**Issue by alberpilot** [wazuh/wazuh#15246]
|Wazuh version|Install type|Action performed|Platform|
|---|---|---|---|
| 4.3.9-1 | Agent | Packages | Amazon Linux 2 / Windows 2019 |

### Description
Occasionally it is possible to have weird Wazuh agent information using `agent_control -i`. After an automated deployment of 1 Wazuh manager (v4.3.9 debug) and 20 Windows agents (v4.3.8), all the agents have the following `agent_control -i` output:
```
Wazuh agent_control. Agent information:
   Agent ID:   020
   Agent Name: EC2AMAZ-1Q91A5E
   IP address: any/any
   Status:     Active

   Operating system:    Microsoft Windows Server 2019 Datacenter
   Client version:      Wazuh v4.3.8
   Configuration hash:  b34f7b79f00c58815f05603c7fe84570
   Shared file hash:    74c5f8eadd7b67cc7dce2c28ee5872ea
   Last keep alive:     1666799244

   Syscheck last started at:  Wed Oct 26 15:46:34 2022
   Syscheck last ended at:    Wed Oct 26 15:46:56 2022
```
but one of them has this output:
```
Wazuh agent_control. Agent information:
   Agent ID:   019
   Agent Name: EC2AMAZ-156SLH8
   IP address: any/any
   Status:     Active

   Operating system:    Microsoft Windows Server 2019 Datacenter
   Client version:      Wazuh v4.3.8 [Ver: 10.0.17763] - Wazuh v4.3.8
   Configuration hash:  b34f7b79f00c58815f05603c7fe84570
   Shared file hash:    74c5f8eadd7b67cc7dce2c28ee5872ea
   Last keep alive:     1666799226

   Syscheck last started at:  Wed Oct 26 15:46:17 2022
   Syscheck last ended at:    Wed Oct 26 15:46:37 2022
```

The client version is different between both. Both VMs are deployed at the same time, with the same deployment script:
```
Invoke-WebRequest -Uri https://packages.wazuh.com/4.x/windows/wazuh-agent-4.3.8-1.msi -OutFile ${env:tmp}\wazuh-agent-4.3.8.msi
msiexec.exe /i ${env:tmp}\wazuh-agent-4.3.8.msi /q WAZUH_MANAGER='172.31.36.76' WAZUH_REGISTRATION_SERVER='172.31.36.76' WAZUH_AGENT_GROUP='windows,default'
Start-Sleep -Seconds 30
Start-Service -Name WazuhSvc
```
All is supposed the same. 

### Environment 
- 1 Wazuh manager v4.3.9 (Amazon Linux 2)
- 20 Windows agents v4.3.8 (Windows server 2019). (`ami-050d0a1abe93f4773`)

### Steps 
Take into account that is rarely produced. The described steps could not reproduce the described error in all the attempts. 
1. Install Wazuh manager debug v4.3.9
2. Install Wazuh windows agent with the previous script. 

### Evidences
There was not possible to retrieve the `global.db` data due to the automated upgrade test. After the upgrade, the field `Client version` had the same other agent formats. 

## New information added (edit)
It was possible to reproduce this error. 
```
[root@ip-172-31-47-252 ec2-user]# /var/ossec/bin/agent_control -i 019

Wazuh agent_control. Agent information:
   Agent ID:   019
   Agent Name: EC2AMAZ-Q5T3BG8
   IP address: any/any
   Status:     Active

   Operating system:    Microsoft Windows Server 2019 Datacenter
   Client version:      Wazuh v4.3.9Microsoft Windows Server 2019 Datacenter [Ver: 10.0.17763] - Wazuh v4.3.9
   Configuration hash:  b34f7b79f00c58815f05603c7fe84570
   Shared file hash:    74c5f8eadd7b67cc7dce2c28ee5872ea
   Last keep alive:     1666859731

   Syscheck last started at:  Thu Oct 27 06:06:27 2022
   Syscheck last ended at:    Thu Oct 27 06:06:42 2022
```

**Global DB**
```
[root@ip-172-31-47-252 ec2-user]# sqlite3 /var/ossec/queue/db/global.db "select * from agent where id=019;"
19|EC2AMAZ-Q5T3BG8|172.31.37.240|any|c6a233c3f2ba7ec06609c6408e565c9a443f31d05628087313ae61f4e65b8909|Microsoft Windows Server 2019 Datacenter|10.0.17763|10|0||17763|windows|Microsoft Windows Server 2019 Datacenter||Wazuh v4.3.9Microsoft Windows Server 2019 Datacenter [Ver: 10.0.17763] - Wazuh v4.3.9|b34f7b79f00c58815f05603c7fe84570|74c5f8eadd7b67cc7dce2c28ee5872ea|ip-172-31-47-252.ec2.internal|node01|1666807423|1666860061|windows,default|synced|active|0
```
**Manager ossec.log: `remoted` messages**
Error not found. 
```
[root@ip-172-31-47-252 ec2-user]# zcat /var/ossec/logs/wazuh/2022/Oct/ossec-26.log.gz | egrep -i remoted
2022/10/26 18:00:40 wazuh-remoted: INFO: Started (pid: 3988). Listening on port 1514/TCP (secure).
2022/10/26 18:00:40 wazuh-remoted: INFO: (1410): Reading authentication keys file.
2022/10/26 18:03:20 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
2022/10/26 18:03:20 wazuh-remoted: INFO: (1410): Reading authentication keys file.
2022/10/26 18:03:30 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
2022/10/26 18:03:30 wazuh-remoted: INFO: (1410): Reading authentication keys file.
2022/10/26 18:03:40 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
2022/10/26 18:03:40 wazuh-remoted: INFO: (1410): Reading authentication keys file.
2022/10/26 18:03:50 wazuh-remoted: INFO: (1409): Authentication file changed. Updating.
2022/10/26 18:03:50 wazuh-remoted: INFO: (1410): Reading authentication keys file.
```
**Windows agent ossec.log: printed version**
```
2022/10/26 18:04:03 wazuh-agent: INFO: Windows version is 6.0 or newer. (Microsoft Windows Server 2019 Datacenter [Ver: 10.0.17763] - Wazuh v4.3.8).
2022/10/26 18:06:07 wazuh-agent: INFO: Windows version is 6.0 or newer. (Microsoft Windows Server 2019 Datacenter [Ver: 10.0.17763] - Wazuh v4.3.9).
```

**Comment by vikman90** [comment]
Possibly related to #12731.

**Comment by vikman90** [comment]
Possibly related to #12731.

**Comment by davidjiglesias** [comment]
Already fixed in #12731
